app/Services/Adselect.php

```diff
@@ -51,7 +49,7 @@ public function getStatus()
 
     public function addImpressions(array $events)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$events". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)